start-here/js/script-main.js

```diff
@@ -18,10 +18,70 @@ function handleSearch() {
     input.classList.add("search-box--warning");
   } else {
     warningMessage.style.display = "none";
+    fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${input.value}`)
+      .then((response) => response.json())
+      .then((data) => {
+        if (data.title) {
```

Comment:
I couldn't see any results coming from lines 24 to 27. Looks like `data.title` is resulting in `undefined`.

Reply:
maybe it's value you inputed to search

Reply:
I'm sorry for the lack of content.
On the image below is the code I can't see the result.
<img width="758" alt="Screenshot 2024-07-21 at 20 30 22" src="https://github.com/user-attachments/assets/0b205288-70eb-4234-be7f-a2704d0fa0bc">

If I type a nonsense word and click on the search icon, I think it is meant to have a message alert for the user, so he can understand what type of failure just happened.
Here is what comes out of my console.log at line 23 of the preview image:
![Screenshot 2024-07-21 at 20 33 55](https://github.com/user-attachments/assets/6731f305-062b-40f5-b0e2-dbfcc8b716fd)